index.html

```diff
@@ -19,7 +19,7 @@
 <body>
     <nav>
         <div class="container0">
-            <a href="#" class='logo'>My Logo</a>
+            <a href="#" class='logo'>AD</a>
```

Comment:
- Good job using an `a` tag for your logo 👍 👍, but please ensure it navigates to the homepage when it is clicked on, this helps to improve the user experience on your site